### ISSUE TITLE
Control whether GSMTC aka media controls in media flyout would show or not

### ISSUE THREAD

**Issue by 01Sub01** [ModernFlyouts-Community/ModernFlyouts#1092]
### Description of the new feature / enhancement

An option will be toggled to enable/disable whether or not Media Controls OSD would show when either the Play/Pause, Forward, Backward buttons are pressed.

### Scenario when this would be used?

When gaming while listening to music, sometimes the media controls OSD would cover up a chunk of the screen which makes it inconvenient so if there was an optional option to disable the media controls OSD when media buttons are pressed, that would be great!

### Additional context

![image](https://user-images.githubusercontent.com/74735825/190882141-2be0f0d0-7b08-4a2f-89cb-c94361c979d6.png)
Basically a toggle option for: 
Show Media Control in System Media Transport Controls (GSMTC) aka Media controls in Media Flyout

**Comment by Samuel12321** [comment]
Hi @01Sub01 ,
can you clarify what you mean? There is already a way of disabling the media flyout?

**Comment by Samuel12321** [comment]
Ok i get it now, you want to be able to completely disable the media flyout.
This will be in 0.11.
duplicate of #719 
Additionally we are looking into disabling while in full screen mode #99
